test/widgets/autocomplete_test.dart

```diff
@@ -108,24 +108,26 @@ void main() {
   group('ComposeAutocomplete', () {
 
     Finder findNetworkImage(String url) {
+      String trimmedUrl = url.replaceAll(RegExp(r'\.(png|jpg|jpeg|gif|bmp|webp)$',caseSensitive: false), ''); //Remove the extension from the URL
       return find.byWidgetPredicate((widget) => switch(widget) {
-        Image(image: NetworkImage(url: var imageUrl)) when imageUrl == url
-          => true,
+        Image(image: NetworkImage(url: var imageUrl)) when imageUrl.startsWith(trimmedUrl)
+        => true,
```

Comment:
nit: the indentation on the `=> true,` line helps with readability; don't remove it:

```suggestion
        Image(image: NetworkImage(url: var imageUrl)) when imageUrl.startsWith(trimmedUrl)
          => true,
```